server/src/main/resources/rules/rules.js

```diff
@@ -2598,6 +2598,7 @@ var Autobind = {
                 if (role == null) {
                     return null;
                 }
+                role = role.trim();
```

Comment:
Let's move this up before the initial null check, and then turn the null check into a null-or-empty check.

Reply:
Now added a null and empty check on a role. Trim gives an exception on a null value. Hence the first null is checked further the empty string.